Documentation/Signoffs/SensorSysem.md

```diff
@@ -0,0 +1,106 @@
+are more common, but current signals are faster and less likely to lose current or voltage drops in the signal process. Current analog sensors have an extra step where 
+a built-in transmitter in the component takes the voltage and sets a current range for the signal to be generated. Once the current signal gets to the microcontroller, 
+it would need to go back to its voltage via a load resistor. The team plans on getting various types of sensors for the design process of this subsystem so that the 
+user of the Air Quality Device will be able to input various sensors that will accurately state the parameter value.
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner.  The constraints given below 
+are generalized to allow for comparison of any potential contender for the sensors.  Constraints on the subsystem are as follows:
+
+1. Shall have a selection of tested sensors to cover a range of pollutants, particulates, air composites, and other qualities.  Common qualities to measure are carbon 
+monoxide, lead, nitrogen oxides, ozone, particulate matter, and sulfur dioxide.  Other qualities may be selected in the future, but these are definite qualities to 
+look for.  These sensors should work independently from each other.
+
+2. Shall be modular. Specifically, the change of sensor should be seamless and handled by the board and software.
+
+3. Shall utilize sensors that are interchangeable.  The sensors will have multiple kinds of input/output (I/O) ports, so a list will be made that acknowledges all I/O 
+ports of sensors to be approved and tested.  The I/O ports that are shared among a group of sensors should be able to operate seamlessly if swapped out from the same 
+type of I/O port for another sensor in the group using that I/O port.  Sensors that do not share this I/O port will have a port of its own grouping that has this same 
+functionality within that group.
+
+4. Each sensor must be able to communicate with the microcontroller using a communicative language such as Serial Peripheral Interface (SPI), or Universal 
+Asynchronous Receiver-Transmitter (UART). Being able to talk with the sensors is crucial towards the pseudo code created in the Microcontroller subsystem.
+
+5. Shall be able to operate with at least 3 sensors sampling independently of each other.  Sensors will take samples at set intervals and be read and stored on the 
+device periodically. Samples may be received and outputted one at a time.
+
+6. The speed for sample rates when receiving analog signal input from the sensors should be sampled at a rate in the range of 44.1kHz - 48 kHz. Having sample rates 
+that are too low causes data to be lost in the data calibration process.
+
+7. The input voltages from the interface will provide either 3 or 5 V DC. Therefore, each sensor that will be used for the device shall have input voltages in that 
```

Comment:
First sentence is not clear. Second sentence is fluff. Not sure what third sentence is trying to say.